packages/core/src/login.ts

```diff
@@ -63,15 +67,17 @@ export function getIdInfoFromApiKey(
   httpClient: CDFHttpClient,
   apiKey: string
 ): Promise<null | IdInfo> {
-  return getIdInfo(httpClient, { [API_KEY_HEADER]: apiKey });
+  return getIdInfo(httpClient.get.bind(httpClient), {
+    [API_KEY_HEADER]: apiKey,
+  });
 }
 
 /** @hidden */
 export function getIdInfoFromAccessToken(
   httpClient: CDFHttpClient,
   accessToken: string
 ): Promise<null | IdInfo> {
-  return getIdInfo(httpClient, {
+  return getIdInfo(httpClient.get.bind(httpClient), {
```

Comment:
this comes all the way from `createAuthenticateFunction` -> `Login.loginSilently(httpClient, ...)` maybe do it there instead?

Reply:
unrelated